studies/models.py

```diff
@@ -1470,9 +1470,7 @@ def from_pipe_payload(cls, pipe_response_dict: dict):
         return cls.objects.create(
             pipe_name=old_pipe_name,
             pipe_numeric_id=data["id"],
-            s3_timestamp=datetime.fromtimestamp(
-                int(timestamp) / 1000, tz=datetime.timezone.utc
-            ),
+            s3_timestamp=datetime.fromtimestamp(int(timestamp) / 1000, tz=timezone.utc),
```

Comment:
I think this is the reason that the webhook is still failing on the staging site.